packages/@robojs/cron/src/core/cron.ts

```diff
@@ -103,9 +101,15 @@ export function Cron(cronExpression: string, jobFunction: string | (() => void))
 	return new CronJob(cronExpression, jobFunction)
 }
 
+Cron.get = (id: string): CronJob | null => {
+	return getState(`${NAMESPACE}${id}`)
+}
+
 Cron.remove = async (id: string): Promise<void> => {
-	await Flashcore.delete(id, { namespace: '__plugin_cron_' })
+	await Flashcore.delete(id, { namespace: NAMESPACE })
 	await Flashcore.set('jobs', (jobs: string[] = []) => jobs.filter((jobId) => jobId !== id), {
-		namespace: '__plugin_cron_'
+		namespace: NAMESPACE
 	})
+
+	setState(`${NAMESPACE}${id}`, null)
```

Comment:
Don't forget you can also apply namespaces using a third parameter. (or second in `getState`)
https://robojs.dev/robojs/state#namespaces